charts/linkerd2/Chart.yaml

```diff
@@ -1,9 +1,8 @@
 apiVersion: "v1"
 # this version will be updated by the CI before publishing the Helm tarball
 appVersion: edge-XX.X.X
-description: |
-  Linkerd gives you observability, reliability, and security
-  for your microservices — with no code change required.
+deprecated: true
+description: "DEPRECATED: Use linkerd-crds and linkerd-control-plane instead - Linkerd gives you observability, reliability, and security for your microservices — with no code change required."
```

Comment:
```suggestion
description: "DEPRECATED: Use linkerd-crds and linkerd-control-plane instead - Linkerd gives you observability, reliability, and security for your microservices — with no code change required."
```

We haven't published these for 2.11 right? Should we say something like: "Use linkerd-crds and linkerd-control-plane for Linkerd 2.12 and later. See <upgrade notes link>"

Reply:
Ah right, those charts for relevant only for 2.12.0 and later. I'll change the wording.

Reply:
Pending on linikerd/website#1494 which is modifying the upgrade doc titles, to provide an exact link.